examples/prometheus/prometheus.yaml

```diff
@@ -139,6 +147,58 @@ objects:
           - mountPath: /prometheus
             name: prometheus-data
 
+        # Deploy alertmanager behind prometheus-alert-buffer behind an oauth proxy
+        - name: alerts-oauth-proxy
+          image: ${IMAGE_PROXY}
+          imagePullPolicy: IfNotPresent
+          ports:
+          - containerPort: 9443
+            name: web
+          args:
+          - -provider=openshift
+          - -https-address=:9443
+          - -http-address=:4190
```

Comment:
Since the 2 proxies are in the same pod, and oauth-proxy listens on http with default port 4180, I had to change the port on the second proxy so they don't collide.